crates/node/src/interactive_cli/identity.rs

```diff
@@ -44,8 +44,8 @@ enum AliasSubcommands {
         name: Alias<PublicKey>,
         /// The identity to create an alias for
         identity: PublicKey,
-        /// The context that the identity is a member of (omit to use default context)
-        #[clap(long = "context", short = 'c')]
+
+        #[clap(long, short, default_value = "default")]
         context: Option<Alias<ContextId>>,
```

Comment:
```suggestion
        #[clap(long, short, default_value = "default")]
        context: Alias<ContextId>,
```

with `default_value` we don't need `Option`